crowbar_framework/app/assets/javascripts/bootstrap/growl.js

```diff
@@ -0,0 +1,186 @@
+		}
+
+		if (options.template.message) {
+			$growl.append($(options.template.message).html(message));
+		}else{
+			$growl.append(message);
+		}
+
+		/* ===== DETERMINE GROWL POSITION ===== */
+		offsetAmount = options.offset;
+
+		$("."+growlClass).each(function() {
+			return offsetAmount = Math.max(offsetAmount, parseInt($(this).css(options.position.from)) + $(this).outerHeight() + options.spacing);
+		});
+
+		css = {
+			"position": (options.ele === "body" ? "fixed" : "absolute"),
+			"margin": 0,
+			"z-index": options.z_index,
+			"display": "none"
+		};
+
+		css[options.position.from] = offsetAmount + "px";
+		$growl.css(css);
+		$(options.ele).append($growl);
+
+		switch (options.position.align) {
+			case "center":
+				$growl.css({
+					"left": "50%",
+					"marginLeft": -($growl.outerWidth() / 2) + "px"
+				});
+				break;
+			case "left":
+				$growl.css("left", options.offset + "px");
+				break;
+			case "right":
+				$growl.css("right", options.offset + "px");
+				break;
+		}
+
+		/* ===== DETERMINE GROWL POSITION ===== */
+		if (options.onGrowlShow) {
+			options.onGrowlShow(event);
+		}
+
+		var fadeIn = $growl.fadeIn(options.fade_in, function(event) {
+			if (options.onGrowlShown) {
+				options.onGrowlShown(event);
+			}
+
+			/* ===== HANDEL DELAY AND PAUSE ON MOUSE OVER ===== */
+			if (options.delay > 0) {
+				if (options.pause_on_mouseover == true) {
+					$growl.on('mouseover', function() {
+						clearTimeout(bootstrap_growl_remove[$growl.index()]);
+					}).on('mouseleave', function() {
+						bootstrap_growl_remove[$growl.index()] = setTimeout(function() {
+							return $growl.alert("close");
+						}, options.delay);
+					});
+				}
+
+				bootstrap_growl_remove[$growl.index()] = setTimeout(function() {
+					return $growl.alert("close");
+				}, options.delay);
+			}
+		});
+
+		$growl.bind('close.bs.alert', function (event) {
+			if (options.onGrowlClose) {
+				options.onGrowlClose(event);
+			}
+		});
+
+		$growl.bind('closed.bs.alert', function (event) {
+			if (options.onGrowlClosed) {
+				options.onGrowlClosed(event);
+			}
+
+			var pos = $(this).css(options.position.from);
+			$(this).nextAll('.'+growlClass).each(function() {
+				$(this).css(options.position.from , pos);
+				pos = (parseInt(pos)+(options.spacing)) + $(this).outerHeight();
+			});
+		});
+
+		return $growl;
+
+	};
+
+	$.growl.default_options = {
+		ele: "body",
+		type: "info",
+		allow_dismiss: true,
+		position: {
+			from: "top",
+			align: "right"
+		},
+		offset: 20,
+		spacing: 10,
+		z_index: 1031,
+		fade_in: 400,
+		delay: 5000,
+		pause_on_mouseover: false,
+		onGrowlShow: null,
+		onGrowlShown: null,
+		onGrowlClose: null,
+		onGrowlClosed: null,
+		template: {
+			icon_type: 'class',
+			container: '<div class="col-xs-10 col-sm-10 col-md-3 alert">',
+			dismiss: '<button type="button" class="close" data-dismiss="alert" aria-hidden="true">&times;</button>',
+			title: '<strong>',
+			title_divider: '',
```

Comment:
Identifier 'title_divider' is not in camel case.<br>Mixed double and single quotes.